backend/ownerSizeTotalWorker.js

```diff
@@ -149,10 +149,27 @@ sbp('sbp/selectors/register', {
 
     // Map to store: ultimateOwnerID -> [totalDelta, Set<originalResourceIDs>]
     const ultimateOwners = new Map()
+    const orphansSet = new Set()
     // Phase 1: Find the ultimate owner for each resource and aggregate deltas.
     await Promise.all(deltaEntries.map(async ([contractID, delta]) => {
       // --- Potentially Slow Owner Lookup Loop ---
-      const ownerID = cachedUltimateOwnerMap.get(contractID) || await lookupUltimateOwner(contractID)
+      const cachedOwnerID = cachedUltimateOwnerMap.get(contractID)
+      const ownerID = cachedOwnerID || await lookupUltimateOwner(contractID)
+      // If the delta update is for an orphaned resource, it likely was
+      // deleted and needs special handling
```

Comment:
Could you explain in the comment exactly what an "orphaned resource" is and how it was generated?